### PR TITLE
CGPROD-483_Example-gameplay-component

### PR DESCRIPTION
**Pull request by Leonie-**
Adds example gameplay component.

### CODE REVIEW

File: src/components/balloon-pump-game.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import { Screen } from "../../node_modules/genie/src/core/screen.js";
+
+export class BalloonPumpGame extends Screen {
+    constructor() {
+        super();
+    }
+
+    preload() {
+        this.keyLookup = this.layoutFactory.keyLookups[this.game.state.current];
+        // this.selectedSeason = this.context.inState.transient.option && this.context.inState.transient.option === 1 ? "winter" : "summer";
```
Comment (SeanHolden) [inline src/components/balloon-pump-game.js]: can this be removed now?

File: src/main.js [hunk 2]
```diff
@@ -1,29 +1,58 @@
...
+import { BalloonPumpGame } from "./components/balloon-pump-game.js";
+import { Results } from "../node_modules/genie/src/components/results.js";
 import { startup } from "../node_modules/genie/src/core/startup.js";
 
 const transitions = [
     {
         name: "loadscreen",
         state: new Loadscreen(),
         nextScreenName: () => "home",
+        // nextScreenName: () => "game",
```
Comment (SeanHolden) [inline src/main.js]: Can this be removed now?

File: src/components/balloon-pump-game.js (hunk 1 above)
Comment (Leonie-) [reply src/components/balloon-pump-game.js]: I left it in in case we need to remember how to wire up the select screen. Happy to take it out if you like.

**Comment by Thomas-James-Rose** [comment]
Had a play around with this locally and it looks good! One minor observation is that I've noticed the timer text in the game component seems to go off screen when you shrink the viewport. Other than that (and the Genie Core bug fixes still in progress), it seems to be working great! 👍

File: src/main.js (hunk 2 above)
Comment (Leonie-) [reply src/main.js]: I left it in as it makes it easy when you're developing a game to shortcut straight to the game itself. But it's not super important we leave it in.

**Comment by Thomas-James-Rose** [comment]
I've also noticed that it still seems to possible to inflate the balloon when on the pause menu. The time will still be frozen but if you complete the game this way it takes you straight through to the results screen when you hit the play button again.

**Comment by Leonie-** [comment]
@Thomas-James-Rose - good spot, will investigate.

**Comment by Leonie-** [comment]
Just chatted to Sean and the issue with being able to click the balloon on pause is fixed in core 😺

**Pull request merged**